src/main/rx/expresslrs_common.c

```diff
@@ -53,18 +53,20 @@ static uint32_t freqSpread = 0;
 elrsModSettings_t airRateConfig[][ELRS_RATE_MAX] = {
 #ifdef USE_RX_SX127X
```

Comment:
USE_RX_SX127X and USE_RX_SX1280 can't be defined at same time?

Reply:
Can they? Ask the original author 4 years ago!